en/middleware/uxrce_dds.md

```diff
@@ -249,7 +249,23 @@ uxrce_dds_client start -t udp -p 8888 -h 192.168.0.100 -n drone
 ```
 Options `-p` or `-h` are used to bypass `UXRCE_DDS_PRT` and `UXRCE_DDS_AG_IP`.
 
+#### Starting the client in simulation
```

Comment:
We use all letter capitalized headings. Sometimes I think the wrong choice, but we're stuck with it now.

```suggestion
#### Starting the Client in Simulation
```